src/test/java/duke/ParserTest.java

```diff
@@ -0,0 +1,41 @@
+package duke;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+import duke.commands.Command;
+
+
+public class ParserTest {
+    private Parser parser = new Parser();
+    @Test
+    public void doneCommand_exceptionThrown() {
+        try {
+            Command command = parser.findCommand("done five");
+        } catch (DukeException e) {
+            assertEquals(e.getMessage(), "Hi my friend, you need to give a number.");
+        }
+    }
+
+    @Test
+    public void todoCommand_exceptionThrown() {
+        try {
+            Command command = parser.findCommand("todo");
+        } catch (DukeException e) {
+            assertEquals(e.getMessage(), "Hi my friend, you don't even know what you want to do.");
+        }
+    }
+
+    @Test
+    public void wrongCommand_exceptionThrown() {
```

Comment:
The naming for the test methods can be clearer. For example, `doneCommand_exceptionThrown` tests for an input that does not give a proper numerical input, so `doneCommand_noNumberGiven_exceptionThrown` could be a better name under the `featureUnderTest_testScenario_expectedBehavior()` format outlined in the coding standard. Although the standard does allow for the omission of the 2nd and 3rd parts, these tests may not be exhaustive enough to justify dropping the test case section of the name, especially if other exceptions may arise from entering the command in different ways. The other test classes also have a similar problem with method naming.